auth-web/src/routes/router.ts

```diff
@@ -324,18 +330,14 @@ export function getRoutes (): RouteConfig[] {
   return routes
 }
 
-// Feature Flagging HomeView Components
-// For Development only
-const flagCondition = LaunchDarklyService.getFlag('incorporations-launch-feature')
-
 // Get the correct Homeview depending on Environment
 const getEnvHomeView = () => {
-  return flagCondition ? HomeViewDev : HomeView
+  return getFeatureFlag ? HomeViewDev : HomeView
```

Comment:
`getFeatureFlag` is a function not a getter.
This ternary will always return the first response since the function always exists.
Ie, you need () to call the function, not test its existence.

Reply:
Or change the function to a getter.

Reply:
good catch, thanks. I didn't update this when i refactored!